libcontainerd/remote_daemon_options_unix.go

```diff
@@ -1,4 +1,4 @@
-// +build linux solaris
+// +build linux
```

Comment:
this file should probably be renamed to `_linux.go` then the build-tag can be removed

Reply:
Thanks that works! Updated.